### ISSUE TITLE
Github Releases deploy is broken

### ISSUE THREAD

**Issue by tmortiboy** [SEEK-Org/evaporate#24]
As part of a tag build, the deploy to github is broken. I wonder if this is due to the oath token being expired, or due to our new authentication.

Luckily we are still deploying to docker first, so new tags appear there.

**Comment by tmortiboy** [comment]
For reference https://travis-ci.org/seek-oss/evaporate/jobs/385033955#L1124

**Comment by askoufis** [comment]
Could it have something to do with SSO?

**Comment by tmortiboy** [comment]
Yeah most likely. I just don't have time to look into it right now 😢
